mod/content/spec/set/type/asset_file_spec.rb

```diff
@@ -1,9 +1,9 @@
 # -*- encoding : utf-8 -*-
 
-RSpec.describe Card::Set::Type::AssetFile do
-  it"loads content from file" do
-    path = File.join(__dir__ , "asset_file.txt")
-    card = Card.new name: "test", type_id: Card::AssetFileID, content: path
-    expect(card.content).to eq "test content\n"
-  end
-end
+# RSpec.describe Card::Set::Type::AssetFile do
```

Comment:
This was causing issues. Looks like maybe a test from a previous spike when you considered making AssetFile its own type?